### DISCUSSION TITLE
RetroPie config?

### DISCUSSION THREAD

**Discussion by kage1414** [retronas/discussions#21] [Q&A] [answered]
Is there an option for configuring retronas with retropie/arch?

Right now I'm using the emuelec config and it works well, but I'm starting to run into issues with permissions that may be solvable with a separate config?

**Comment by sairuk** [comment, marked as answer]
there is not currently an option but if someone wants to do the config they can open a pull request against the testing branch to add support for that project. The field is there in `retronas_systems.yml` and needs to be populated, then the emuelec installer can be used as a template for adding support.

they have no way to monitor for updates unlike emulelec etc that publish their full configs or gitrepo info because iirc they generate their emulation station config from a script.

I normally use https://github.com/sairuk/retronas-check-systems to monitor for changes
